src/saml2/response.py

```diff
@@ -1044,7 +1044,7 @@ def verify(self, keys=None):
             logger.error("Verification error on the response: %s", err)
             raise
         else:
-            if res is None:
```

Comment:
the validation was good and it simply valud res as False, so this check didn't intecept this